src/features/chains/queries/useScrapedChains.ts

```diff
@@ -52,7 +52,7 @@ export function useScrapedChains(multiProvider: MultiProvider) {
 }
 
 // TODO: Remove once all chains in the DB are scraped
-export function isUnscrapedDbChain(multiProvider: MultiProvider, chainIdOrName: number | string) {
-  const chainName = multiProvider.tryGetChainName(chainIdOrName);
+export function isUnscrapedDbChain(multiProvider: MultiProvider, domainId: DomainId) {
```

Comment:
had to change a bunch of `chainIdOrName` because it's now just `domainId` or `chainName` in places